src/templates/Page/ReleaseNotesArchiveList.tsx

```diff
@@ -0,0 +1,86 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import TableOfContents from './TableOfContents';
+
+type ProductSlug = 'geoip' | 'minfraud';
+
+export interface IReleaseNotesArchiveList {
+  type: ProductSlug;
+}
+
+const years: {[index: string]: number[]} = {
```

Comment:
Not a huge deal, but I think a type of `Record<ProductSlug, number[]>` here might help out intellisense/autocomplete a bit more.

Reply:
Ah, good call.  Forgot about `Record<>`